brands/shop/shoes.json

```diff
@@ -294,6 +294,14 @@
       "shop": "shoes"
     }
   },
+  "shop/shoes|Foot Solutions": {
+    "tags": {
+      "brand": "Foot Solutions",
+      "brand:wikidata": "Q69982369",
+      "name": "Foot Solutions",
+      "shop": "shoes"
```

Comment:
> I'm not sure how to add this to "healthcare" as I haven't found this label.

You can add any tags you need to as properties in this JSON object, alongside `shop`, `name`, etc. I was suggesting adding a `healthcare=something` or `healthcare:speciality=something` yah but wasn’t sure exactly what to set it to. Later on, as I fleshed out the Foot Solutions item on Wikidata to save it from deletion, I learned of the word “pedorthist”, which would work as a value for the `healthcare:speciality` tag.

Reply:
Added in 6a25dbc04ff82a35fd49f0d73b247dc099851d82 along with `shoes=orthopaedic`, which is [documented on the wiki](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dshoes).